alby.go

```diff
@@ -269,6 +270,33 @@ func (svc *AlbyOAuthService) LookupInvoice(ctx context.Context, senderPubkey str
 	return "", false, errors.New(errorPayload.Message)
 }
 
+func (svc *AlbyOAuthService) GetInfo(ctx context.Context, senderPubkey string) (info *NodeInfo, err error) {
+	app := App{}
+	err = svc.db.Preload("User").First(&app, &App{
+		NostrPubkey: senderPubkey,
+	}).Error
+	if err != nil {
+		svc.Logger.WithFields(logrus.Fields{
+			"senderPubkey": senderPubkey,
+		}).Errorf("App not found: %v", err)
+		return nil, err
+	}
+
+	svc.Logger.WithFields(logrus.Fields{
+		"senderPubkey": senderPubkey,
+		"appId":        app.ID,
+		"userId":       app.User.ID,
+	}).Info("Info fetch successful")
+	return &NodeInfo{
+		Alias:       "getalby.com",
```

Comment:
I am always wondering what information is useful here. 
Maybe this should be the lightning address in our case?

Reply:
@bumi that would be great and @stackingsaunter also suggested it. I would vote to make that change in the extension too.

CC @reneaaron

Reply:
I will create a separate issue for this